Source/engine/render/automap_render.cpp

```diff
@@ -1,3 +1,8 @@
+/**
+ * @file automap_render.cpp
+ *
+ * Sprite animation handeling.

```

Comment:
This comment looks wrong. Moreover, why even have file comments for `cpp` files when there is a corresponding header?

Reply:
Doxygen doesn't pickup files unless they have a header :/

Reply:
I suppose we could see this as a feature and only pickup public header functions variables etc?

Reply:
Yeah, I think it's a feature -- the stuff in cpp files is implementation details